### PR TITLE
Added tests for groups claim mapping to roles

### PR DESCRIPTION
**Pull request by honza-kasik**
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [ ] N/A Link to the passing job is provided
- [x] Code is self-descriptive and/or documented
- [x] Description of the tests scenarios is included (see #46)

### CODE REVIEW

**Comment by zhantemirov** [comment]
I've been thinking over possible roles/endpoints combinations - but didn't managed to make up any proper one - all sensible combinations IMO are already presented. 

Tests structure is clear and very understandable, thanks.

LGTM.

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+     * @tpPassCrit The {@code groups} claim is mapped to application server roles and endpoint returns expected
+     *             response with "monitor" role name.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void monitorAccessMonitorPath(@ArquillianResource URL url) {
+        final JsonWebToken token = new JwtHelper(keyTool, "issuer")
+                .generateProperSignedJwt(Collections.singleton(Roles.MONITOR));
+
+        RestAssured.given()
```
Comment (mnovak1) [inline microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/rbac/RolesAllowedRbacTest.java]: Can it be added in to static import, please?

**Comment by mnovak1** [comment]
@honza-kasik just minor change requested, otherwise LGTM

**Comment by honza-kasik** [comment]
@mnovak1 Thanks! I also squashed commits.

**Comment by mnovak1** [comment]
thanks, merged.

**Pull request merged**